### PR TITLE
Remove erroneous palisade gates from two houses

### LINKED ISSUE #28479: Impossible open palisade gate tiles in house
**Describe the bug**  
There's a house with many “impossible” open palidade gate tiles (`t_palisade_gate_o`).
These tiles are placed in such a way that they don't make sense. There are only these gate tiles but no corresponding palidade tiles to which they “attach”.
The way these tiles look like makes me assume these tiles were intended to be normal dirt tiles instead.

This bug is very visible in the Tiles version.

Note: For some reason, the in-game name of the tile is just “Dirt” which seems a bit confusing.

**To Reproduce**  
1. Generate the house until you find the correct one. Sorry, I have no idea how to reproduce it better.

**Expected behavior**  
The house in question should not have any palisade-related tiles whatsoever.

**Screenshots**  
Screenshot of the offending house

The `t_palisade_gate_o` tiles are at the outside, these strange wooden things.

(ignore the sign, it was not part of the house)

**Versions and configuration(please complete the following information):**
 - OS: GNU/Linux
 - Game Version: 0.C-37564-g412ee858b5
 - Graphics version: Both, but it's more visible in the Tiles version.
 - Mods loaded: DDA

**Additional context**  
N/A

### PR DESCRIPTION
**Pull request by Reclusive-reptile**
#### Summary
```SUMMARY: Bugfixes "Remove erroneous palisade gates from two houses"```

#### Purpose of change
Fixes #28479

#### Describe the solution
Replaced the palisade gate terrain with dirt.
#### Describe alternatives you've considered
Leave ugly and wrong mapgen in place I guess.

### CODE REVIEW

File: data/mods/Craft_Gunpowder/cgp_recipes.json [hunk 1]
```diff
@@ -363,7 +363,7 @@
     "book_learn": [ [ "manual_electronics", 4 ], [ "textbook_electronics", 4 ] ],
     "using": [ [ "soldering_standard", 20 ], [ "surface_heat", 20 ] ],
     "qualities": [ { "id": "SCREW_FINE", "level": 1 }, { "id": "CHEM", "level": 2 } ],
-    "tools": [ [ [ "mold_plastic", -1 ] ], [ [ "toolset", -1 ] ] ],
+    "tools": [ [ [ "mold_plastic", -1 ], [ "toolset", -1 ] ] ],
```
Comment (ZhilkinSerg) [inline data/mods/Craft_Gunpowder/cgp_recipes.json]: Some unrelated changes here.

**Comment by Wuzzy2** [comment]
I was also able to find `t_palisade_gate_o` in these files:

* `json/mapgen/house/house_vacant.json` (2 times)
* `json/mapgen/trailer_park.json` (1 time)
